features/step_definitions/pos_steps.rb

```diff
@@ -4,10 +4,18 @@
     case object['type']
     when 'user'
       cart_update = JSON.generate({
-        user_id: object['id']
+        user_id: object['id'],
+        lock_version: @cart_version
       })
       cart = put(cart_path(id: @cart_id, format: :json), cart_update)
-      post(cart_cart_payment_path(cart_id: @cart_id, format: :json))
+      @cart_version = JSON.parse(cart.body)['lock_version']
+      cart_payment = JSON.generate({
+        lock_version: @cart_version
+      })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.